django/db/backends/mysql/operations.py

```diff
@@ -25,6 +25,7 @@ class DatabaseOperations(BaseDatabaseOperations):
         'PositiveSmallIntegerField': 'unsigned integer',
     }
     cast_char_field_without_max_length = 'char'
+    explain_prefix = 'EXPLAIN'
```

Comment:
Need a trailing space here or a leading space here I think.

Reply:
I don't think so, it's used in the `as_sql` function which is a list of tokens, joined with a space